src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Alert Rule Action
+sidebar_order: 3
+description: "Learn about the alert rule action UI component."
+---
+
+The Alert Rule Action component allows the developer to get parameters to define routing or configuration in alert rules for their service. When alert rules are triggered for a user, the configured service will receive [Issue Alert](/product/integrations/integration-platform/webhooks/#issue-alerts) and [Metric Alert](/product/integrations/integration-platform/webhooks/#metric-alerts) webhook events with the specified settings for that service to further route the alert.
```

Comment:
No particular reason for the 3rd person, I think just a side effect of lots of people writing and editing these over the years. I _think_ this should be "get" because the user configures (sets) their settings  in the alert rule UI component, and the developer's services receives (gets) that data and can route based on it.